packages/forma-36-react-components/src/components/Button/Button.css

```diff
@@ -29,48 +25,58 @@
       var(--transition-easing-default);
   vertical-align: middle;
   text-decoration: none;
+  box-shadow: 0px 1px 0px rgb(25 37 50 / 10%);
 
   &:hover:not(.Button--disabled) {
     cursor: pointer;
   }
+
+  &:focus:not(.Button--disabled) {
+    outline: none;
+  }
 }
 
 .Button__icon {
   min-width: calc(1rem * (18 / var(--font-base-default)));
+  margin-right: calc(1rem * (8 / var(--font-base-default)));
+}
+
+.Button__dropdown-icon {
+  margin-left: calc(1rem * (8 / var(--font-base-default)));
 }
 
 .Button--full-width {
   width: 100%;
 }
 
 .Button--muted {
-  background-color: var(--color-element-light);
+  background-color: var(--color-white);
```

Comment:
@gui-santos was there the idea of introducing this change in v4? From my perspective, it would be good to test it now to check the impact, so all good from my side, just something that came to my mind.

Reply:
uhmmm I thought all color changes were v3
I will test in the webapp today, but if anyone else wants to check it too pls do 🙏